flink-table/flink-table-calcite-bridge/pom.xml

```
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<artifactId>flink-table</artifactId>
+		<groupId>org.apache.flink</groupId>
+		<version>1.18-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>flink-table-calcite-bridge</artifactId>
+	<name>Flink : Table : Calcite Bridge</name>
+	<description>
+		This module contains the Calcite dependencies for writing planner plugins (e.g. SQL dialects)
+		that interact with Calcite APIs. More exactly, currently, it is intended to provide the ability
+		to create RelNode, which involves accessing the RelOptCluster, RelBuilder, etc, provided by PlannerContext.
+	</description>
+
+	<dependencies>
+		<dependency>
+			<!-- Used for translation of table programs -->
+			<groupId>org.apache.calcite</groupId>
+			<artifactId>calcite-core</artifactId>
+			<!-- When updating the Calcite version, make sure to update the dependency exclusions -->
+			<version>${calcite.version}</version>
+			<exclusions>
+				<!--
+				"mvn dependency:tree" as of Calcite 1.26.0:
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.26.0:compile
+				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.26.0:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.12.1:compile
+				[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.0:compile
+				[INFO] |  +- com.esri.geometry:esri-geometry-api:jar:2.2.0:runtime
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.12.1:runtime
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.12.1:runtime
+				[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.4.0:runtime
+				[INFO] |  |  \- net.minidev:json-smart:jar:2.3:runtime
+				[INFO] |  |     \- net.minidev:accessors-smart:jar:1.2:runtime
+				[INFO] |  |        \- org.ow2.asm:asm:jar:5.0.4:runtime
+				[INFO] |  +- commons-codec:commons-codec:jar:1.10:runtime
+				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
+				[INFO] |  \- commons-io:commons-io:jar:2.4:compile
```

it should be 1.29.0 dependencies like at https://github.com/apache/flink/pull/22166/files#diff-6d81d722fce4e303d49b2136ae8e137f74ed3f4ece4d2a990445bcf6a2388392L132-L147
```suggestion
								"mvn dependency:tree" as of Calcite 1.29.0:
				[INFO] +- org.apache.calcite:calcite-core:jar:1.29.0:compile
				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.29.0:compile
				[INFO] |  +- com.esri.geometry:esri-geometry-api:jar:2.2.0:compile
				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.13.4:compile
				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.20.0:compile
				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.13.4:compile
				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.13.4.2:compile
				[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.4.0:runtime
				[INFO] |  |  \- net.minidev:json-smart:jar:2.3:runtime
				[INFO] |  |     \- net.minidev:accessors-smart:jar:1.2:runtime
				[INFO] |  |        \- org.ow2.asm:asm:jar:5.0.4:runtime
				[INFO] |  +- commons-codec:commons-codec:jar:1.15:runtime
				[INFO] |  \- commons-io:commons-io:jar:2.11.0:compile
```